src/content/app/genome-browser/state/api/genomeBrowserApiSlice.ts

```diff
@@ -117,8 +117,11 @@ const genomeBrowserRestApiSlice = restApiSlice.injectEndpoints({
   })
 });
 
-export const { getTrackPanelGene, gbRegion: getGBRegion } =
-  genomeBrowserApiSlice.endpoints;
+export const {
+  getTrackPanelGene,
+  gbRegion: getGBRegion,
+  gbVariant: getGBVariant
+} = genomeBrowserApiSlice.endpoints;
```

Comment:
After the changes in focusObjectSlice have been reverted, we don't need this export for now. I'm sure we'll need something like this in the future; but let's add it when we need it.

Reply:
✅